### PR TITLE
Zach sign up page

### LINKED ISSUE #15: heroku down?
as of now (march 9th, 3pm), heroku app is down or not deployed. Or the link given in milestone 1 was incorrect.
snapshot1

### PR DESCRIPTION
**Pull request by ZachDFK**
So can someone else look at this and give a :+1:  

or if something doesn't make sense comment on it

Fixes #15

### CODE REVIEW

File: app/assets/stylesheets/scaffolds.scss [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    margin-bottom: 0px;
+    background-color: #c00;
+    color: #fff;
+  }
+
+  ul li {
+    font-size: 12px;
+    list-style: square;
+  }
+}
```
Comment (kevincox) [inline app/assets/stylesheets/scaffolds.scss]: :paperclip: Add the newline here.

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -2,4 +2,8 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+ def hello
+	render text:"hi bud"
+ end
```
Comment (kevincox) [inline app/controllers/application_controller.rb]: Please indent this properly.

File: app/models/user.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+=end
+
+
+def lookup_username(uname)
+	return User.find_by username: uname
+end
+
+def lookup_email(ema)
+	return User.find_by email: ema
+end
```
Comment (kevincox) [inline app/models/user.rb]: Indent this please!

File: app/models/user.rb [hunk 4]
```diff
@@ -0,0 +1,24 @@
+class User < ActiveRecord::Base
+=begin
+	User attributes
+
+	username: 	this is the username that the user goes by it can be used to log in. 
+	 			It showed be shown next to users posts and comments. S
+	 			Should be unique
+	password: 	personal password used to log in 
+	email: 		user's email addres it can be used to log in
```
Comment (kevincox) [inline app/models/user.rb]: Don't use tabs after content.

File: app/models/user.rb (hunk 3 above)
Comment (kevincox) [inline app/models/user.rb]: Also we should probably pick tabs or spaces.

**Comment by kevincox** [comment]
My only gripes are the formatting, other then that looks good.

File: app/assets/stylesheets/scaffolds.scss (hunk 1 above)
Comment (ZachDFK) [inline app/assets/stylesheets/scaffolds.scss]: where do you want the new line?

Comment (kevincox) [inline app/assets/stylesheets/scaffolds.scss]: At the end of the file. You can see the red icon thing in github because it isn't a valid text file.

Comment (ZachDFK) [inline app/assets/stylesheets/scaffolds.scss]: oh got it, I didn't see it before viewing the full file

**Pull request closed without merging**